<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Quiz Master GUI</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Solution PDF</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="4">
    <widget class="QRadioButton" name="radioButton_6">
     <property name="text">
      <string>Solutions at end of question file</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="6" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>75</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>New Quiz</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QSpinBox" name="spinBox">
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QRadioButton" name="radioButton_3">
     <property name="text">
      <string>Solutions in separate file</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
   </item>
   <item row="5" column="6" colspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Q's PDF</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QRadioButton" name="radioButton_5">
     <property name="text">
      <string>Solutions after the questions</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="4" colspan="4">
    <widget class="QRadioButton" name="radioButton_4">
     <property name="text">
      <string>Solutions marked in the Questions</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
   </item>
   <item row="6" column="4" colspan="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="minimum">
      <double>-9.000000000000000</double>
     </property>
     <property name="maximum">
      <double>0.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>-0.670000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="5" colspan="2">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>Old Quiz</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="5" column="1" colspan="5">
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>75</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Marking Scheme</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="7" column="6" colspan="2">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
